### PR TITLE
migrate design validation

### PR DESCRIPTION
**Pull request by codeSafari10**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ashparshp** [comment]
@codeSafari10 let's discuss this PR during the Meshery dev meeting call, if you would. Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk).

**Comment by leecalcote** [comment]
@captain-Akshay FYI

File: ui/machines/validator/designValidator.js [hunk 1]
```diff
@@ -262,6 +271,7 @@ export const designValidationMachine = createMachine({
       invoke: {
         input: ({ context, event }) => ({ context, event }),
         src: fromPromise(async ({ input }) => {
+          console.log('validate design schema wooo', input);
```
Comment (leecalcote) [inline ui/machines/validator/designValidator.js:274]: Note: item to remove

File: ui/machines/validator/designValidator.js [hunk 2]
```diff
@@ -229,10 +235,13 @@ export const designValidationMachine = createMachine({
...
           const { component } = input.event.data;
-          const def = await getComponentDefinition(component.type, component.model, {
-            apiVersion: component.apiVersion,
+
+          const def = await getComponentDefinition(component.component.kind, component.model.name, {
+            apiVersion: component.component.version,
             annotations: 'include',
           });
+
+          console.log('bundle component for validation', component, def);
```
Comment (leecalcote) [inline ui/machines/validator/designValidator.js:244]: Note: an item to remove

**Pull request merged**